### ISSUE TITLE
about generate training/testset of nuScenes datasest 

### ISSUE THREAD

**Issue by Uk-shin** [w2kun/RNW#4]
HI! Thanks for your great job!
I try to generate training/test set of nuScenes dataset by utilizing the official toobox and dataset (package 01,02,05,09,10).
However, It seems the official toolbox can't directly generate your training setup (i.e., your directory structure, file_list.txt files, etc)
Could you provide your python script file to generate the nuScene training/testing dataset?

**Comment by Ryanuppp** [comment]
> 1 O

hello! I meet the same problem with you. Have you already solved it ?

**Comment by Ryanuppp** [comment]
hello! I met the same problem with you. Have you already solved it ?

**Comment by emilyemliyM** [comment]
> hello! I met the same problem with you. Have you already solved it ?

Hi, Have you solve the problem ? How to get the dataset structure as same as the author?

Thanks.
